mmv1/products/compute/api.yaml

```diff
@@ -13605,6 +13605,68 @@ objects:
               - :ERRORS_ONLY
               - :TRANSLATIONS_ONLY
               - :ALL
+      - !ruby/object:Api::Type::Array
+        name: rules
+        description: 'A list of rules associated with this NAT.'
+        send_empty_value: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Integer
+              name: 'ruleNumber'
+              description: |
+                An integer uniquely identifying a rule in the list.
+                The rule number must be a positive value between 0 and 65000, and must be unique among rules within a NAT.
+              required: true
+              send_empty_value: true
```

Comment:
I wonder if we need `send_empty_value` here. More details about this setting at https://github.com/GoogleCloudPlatform/magic-modules/blob/264d52eb066b66780a0e273329d41bbf66300ab2/mmv1/README.md#apiyaml

Reply:
Are we adding `send_empty_value` here since `0` is a possible value for `rule_number`?